emcc.py

```diff
@@ -1306,6 +1306,7 @@ def get_bitcode_args(input_files):
           args += shared.EMSDK_CXX_OPTS
         if not shared.Building.can_inline():
           args.append('-fno-inline-functions')
+        args += ['-Dunix', '-D__unix', '-D__unix__']
```

Comment:
I think there's already some code in `tools/shared.py` that defines `EMSCRIPTEN` for the wasm backend, this can probably go there.